### ISSUE TITLE
Feature: Add support for asdf's .tool-versions file

### ISSUE THREAD

**Issue by yamadashy** [shivammathur/setup-php#883]
### Describe the feature

Add support for reading PHP version from asdf's `.tool-versions` file in setup-php action. Currently, the action supports `.php-version` file, and adding support for `.tool-versions` would help users who use asdf for version management.

Similar to the current support:
```yaml
- uses: shivammathur/setup-php@v2
  with:
    php-version-file: .tool-versions  # New proposal, similar to .php-version support
```

### Please check the latest release

- [X] I have checked releases, and the feature is missing in the latest patch version of `v2`.

### Underlying issue

Projects using asdf for PHP version management currently need to maintain both `.tool-versions` for local development and either specify the version directly in workflows or maintain a separate `.php-version` file. Supporting `.tool-versions` would allow these projects to use a single source of truth for PHP versions across all environments.

The action already supports reading from `.php-version`, so extending this to support `.tool-versions` would be a natural evolution and help users who prefer asdf for version management.

### Describe alternatives

Current alternatives:
1. Manually specify PHP version in workflow (requires manual sync with `.tool-versions`)
2. Maintain a separate `.php-version` file (requires maintaining two version files)
3. Use a script to generate `.php-version` from `.tool-versions` (adds complexity to workflow)

### Additional context

Example `.tool-versions` file content:
```
php 8.3.13
```

Similar functionality exists in other GitHub Actions:
- actions/setup-node supports `.tool-versions`
```yaml
- uses: actions/setup-node@v4
  with:
    node-version-file: .tool-versions
```

### Are you willing to submit a PR?

Yes

**Comment by shivammathur** [comment]
Added in https://github.com/shivammathur/setup-php/commit/b6d8115f126b1c44da55e2f72380880320cd10cc

**Comment by yamadashy** [comment]
@shivammathur 
Thank you! Looking forward to the release!

**Comment by shivammathur** [comment]
Released in [`2.32.0`](https://github.com/shivammathur/setup-php/releases/tag/2.32.0).
